assets/prototype/application/services/apollo/mutations/useEntityMutation.js

```diff
@@ -14,6 +14,13 @@ const useEntityMutation = (type, id = '') => {
 	const client = useApolloClient();
 	const [result, setResult] = useState({ called: false, loading: false });
 	const mutators = useMutators();
+	let isMounted = true;
+
+	useEffect(() => {
+		return () => {
+			isMounted = false;
+		};
+	});
```

Comment:
Since we might use this kind of check in other future places as well, maybe we can create custom hook for this:
In few cases I noticed that `useIsMounted` hook is using `useRef`:
- https://aceluby.github.io/blog/react-hooks-cant-set-state-on-an-unmounted-component
- https://github.com/adambrgmn/hooks/blob/master/src/use-is-mounted/index.ts
- https://github.com/hupe1980/react-is-mounted-hook/blob/master/src/use-is-mounted.tsx

Reply:
Yes, it can be done in that way as well.

Reply:
I agree with extracting this into a hook

Reply:
created a new issue for doing this:
plz see: https://github.com/eventespresso/event-espresso-core/issues/2046